pandas/core/dtypes/cast.py

```diff
@@ -979,6 +983,11 @@ def convert_dtypes(
     infer_objects : bool, defaults False
         Whether to also infer objects to float/int if possible. Is only hit if the
         object array contains pd.NA.
+    nullable_backend : str, default "pandas"
```

Comment:
How does this interact with the `mode.nullable_backend` option? My guess is folks would expect `nullable_backend` to take precedent if it's specified, but for `mode.nullable_backend` to take precedent if `nullable_backend` isn't given. 

Here's a tiny example of the use case where I'm asking what happens:

```python
with pd.option_context("mode.nullable_backend", "pyarrow"):
    result = expected.convert_dtypes(nullable_backend="pandas")
```

Reply:
This line specifically is a private helper function that backs the public `Series/DataFrame.convert_dtypes`.

The public `Series/DataFrame.convert_dtypes` will not accept a keyword argument `nullable_backend`